packages/prosemirror/README.md

```
@@ -0,0 +1,76 @@
+```
+
+## Contributing
+
+See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
+
+## Contributors ✨
+
+Thanks goes to these incredible people:
+
+<a href="https://github.com/yorkie-team/yorkie-js-sdk/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=yorkie-team/yorkie-js-sdk" />
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)